pages/react.mdx

```diff
@@ -546,21 +553,7 @@ test("Button", () => {
 });
 ```
 
-This creates two snapshots:
```

Comment:
It would be nice to update this example so user knows what to expect.

Reply:
Does it actually still create two snapshots? Afaik it's inlined so only one `toMatchSnapshot()` is needed?